### PR TITLE
Add admin dashboard layout

### PR DESCRIPTION
**Pull request by khang-sun**
## Related Tickets
- [#49373] https://edu-redmine.sun-asterisk.vn/issues/49373

## WHAT (optional)
- Add layouts to admin page

## HOW
- Use bootstrap and css to add and style admin page layout

## Evidence (Screenshot or Video)
image

image

### CODE REVIEW

**Comment by khang-sun** [comment]
ready

File: app/views/layouts/admin.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title><%= full_title yield :title %></title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "admin", media: "all", "data-turbolinks-track": "reload" %>
+    <%= javascript_pack_tag "application", "data-turbolinks-track": "reload" %>
+    <script src="/assets/jquery.js" type="text/javascript"></script>
```
Comment (ngocvt-0484) [inline app/views/layouts/admin.html.erb:10]: xem bỏ đi e nhé

File: app/views/admin/admin_page/home.html.erb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          <td>01-06-2022 18:03</td>
+          <td>Git</td>
+          <td>Checked</td>
+          <td>20</td>
+          <td>15/20</td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</div>
```
Comment (ngocvt-0484) [inline app/views/admin/admin_page/home.html.erb:40]: `<!-- END TODO -->`

**Comment by khang-sun** [comment]
ready

**Pull request merged**